modules/siddhi-core/src/main/java/org/wso2/siddhi/core/SiddhiAppRuntime.java

```diff
@@ -693,4 +694,26 @@ public void enableStats(boolean statsEnabled) {
             log.debug("Siddhi App '" + getName() + "' statistics reporting not changed!");
         }
     }
+
+    /**
+     * To enable and disable Siddhi App playback mode on runtime along with optional parameters.
+     *
+     * @param playBackEnabled whether playback is enabled or not
+     * @param idleTime
+     * @param incrementInMilliseconds
+     */
+    public void enablePlayBack(boolean playBackEnabled, long idleTime, long incrementInMilliseconds) {
```

Comment:
support idleTime, and  incrementInMilliseconds being null so in that case use the default value